client/src/pages/Login.js

```diff
@@ -146,23 +168,27 @@ const Login = ({ isLoginForm }) => {
         <WhiteSpace />
         <WhiteSpace />
         <SubmitButton
-          title={isLoginForm ? "Login" : "Sign Up"}
+          title={isLoginForm ? "Sign In" : "Sign Up"}
           onClick={isLoginForm ? handleLoginWithEmail : handleSignup}
         />
       </form>
       <WhiteSpace />
       {isLoginForm ? (
         <>
           <p>
-            <Link to="/auth/forgot-password">Forgot password?</Link>
+            <TermsLink href="/auth/forgot-password">Forgot password?</TermsLink>
```

Comment:
For `<TermsLink>` I think we need just a big more styling to match Figma (need to be bolder, correct font-family)

Figma:
<img width="170" alt="Screen Shot 2020-05-03 at 9 40 36 PM" src="https://user-images.githubusercontent.com/42228386/80936602-bb8dfc00-8d86-11ea-8040-a7d1f944f766.png">